style.css

```diff
@@ -1,19 +1,42 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
```

Comment:
I see what you want to achieve here. Take a look at https://necolas.github.io/normalize.css/ which is a standard way of doing this :)